internal/app/init/shutdown.go

```diff
@@ -22,25 +22,18 @@ const (
 func listenForPowerButton() (err error) {
 	// Get the acpi_event family.
 
-	genconn, err := genetlink.Dial(nil)
+	conn, err := genetlink.Dial(nil)
 	if err != nil {
 		return err
 	}
 	// nolint: errcheck
-	defer genconn.Close()
-	var f genetlink.Family
-	if f, err = genconn.GetFamily(acpiGenlFamilyName); os.IsNotExist(err) {
```

Comment:
Ahh I see it now. @mdlayher The linting fails because this PR removes the `os` packages and it is still imported. I think you just need to remove `"os"`.

Reply:
Wow, I am sorry at the oversight. I'm trying out gopls in VSCode and it was definitely acting funny in this repo. I haven't had to think about imports in years thanks to automated tools! I'll remove it in the morning.

Reply:
Same :D. No rush.

Reply:
Oh just a reminder. You will need to rebase. We enforce PRs being up to date. I will hold off on merging anything else after you update.